[RA2 Ch2] CPU-related language corrections (#2037)
cntt-n/CNTT: doc/ref_arch/kubernetes/chapters/chapter02.md

| [4.2.5](../../../ref_model/chapters/chapter04.md#425-cloud-infrastructure-profile-capabilities-mapping) | e.cap.015 | SmartNICs | Not required | Optional | |
| [4.2.5](../../../ref_model/chapters/chapter04.md#425-cloud-infrastructure-profile-capabilities-mapping) | e.cap.016 | FPGA/other Acceleration H/W | Not required | Optional | |
| [4.2.5](../../../ref_model/chapters/chapter04.md#425-cloud-infrastructure-profile-capabilities-mapping) | *e.cap.017* | *Ability to monitor L2-L7 data from workload* | *n/a<sup>(3)</sup>* | *n/a<sup>(3)</sup>* | |

> | [4.2.5](../../../ref_model/chapters/chapter04.md#425-cloud-infrastructure-profile-capabilities-mapping) | i.cap.014 | Specifies the proportion of CPU cores consumed by the Cloud Infrastructure system on the worker nodes. If SMT is used, it indicates the number of consumed SMT threads. | 2 | 2 | |

Reviewer: The wording needs some clarification:
While for CPU cores it says "proportion" without saying proportion to what, while for SMT threads it says absolute number.

> | [4.2.5](../../../ref_model/chapters/chapter04.md#425-cloud-infrastructure-profile-capabilities-mapping) | i.cap.014 | Specifies the proportion of CPU cores reserved for the Cloud Infrastructure system on the worker nodes. If SMT is used, it indicates the number of consumed SMT threads. | 2 | 2 | |

Reviewer 2: In RM 4.2.5, i.cap.014 is defined as consumed not reserved:

i.cap.014(CPU cores consumed by the Cloud Infrastructure on the worker nodes) | any | any
-- | -- | --

```suggestion
| [4.2.5](../../../ref_model/chapters/chapter04.md#425-cloud-infrastructure-profile-capabilities-mapping) | i.cap.014 | Specifies the proportion of CPU coresconsumed by the Cloud Infrastructure system on the worker nodes. If SMT is used, it indicates the number of consumed SMT threads. | 2 | 2 | |
```
